ext/snmp/tests/snmpset.phpt

```diff
@@ -76,29 +80,44 @@ var_dump((snmpget($hostname, $communityWrite, $oid2, $timeout, $retries) === $ol
 
 echo "More error handing\n";
 echo "Single OID, single type in array, single value\n";
-$z = snmpset($hostname, $communityWrite, $oid1, array('s'), $newvalue1, $timeout, $retries);
-var_dump($z);
-var_dump((snmpget($hostname, $communityWrite, $oid1, $timeout, $retries) === $oldvalue1));
-var_dump((snmpget($hostname, $communityWrite, $oid2, $timeout, $retries) === $oldvalue2));
+try {
+    $z = snmpset($hostname, $communityWrite, $oid1, array('s'), $newvalue1, $timeout, $retries);
+    var_dump($z);
+    var_dump((snmpget($hostname, $communityWrite, $oid1, $timeout, $retries) === $oldvalue1));
+    var_dump((snmpget($hostname, $communityWrite, $oid2, $timeout, $retries) === $oldvalue2));
```

Comment:
Shouldn't the last two var_dumps always stay outside try/catch? As they verify that nothing was changed.